rwslib/tests/test_rwscmd.py

```diff
@@ -13,21 +15,18 @@ def setUp(self):
 
     @httpretty.activate
     def test_version(self):
-
         httpretty.register_uri(
             httpretty.GET, "https://innovate.mdsol.com/RaveWebServices/version",
             status=200,
             body='1.0.0')
 
-        result = self.runner.invoke(rwscmd.rws, ['https://innovate.mdsol.com','version'],
+        result = self.runner.invoke(rwscmd.rws, ['https://innovate.mdsol.com', 'version'],
```

Comment:
Ok, I'm being slow - I assume we're talking about passing a username/password to an unauth'ed endpoint?